### PR TITLE
Refactor ConcurrencyThrottlingUtils class

### PR DESCRIPTION
**Pull request by GDLMadushanka**

### CODE REVIEW

**Comment by Drifftr** [comment]
Shall we add the public jira in the commit description?

File: modules/core/src/main/java/org/apache/synapse/util/ConcurrencyThrottlingUtils.java [hunk 1]
```diff
@@ -47,9 +47,9 @@ public static void decrementConcurrencyThrottleAccessController(MessageContext s
         if (isConcurrencyThrottleEnabled != null && isConcurrencyThrottleEnabled) {
             ConcurrentAccessController concurrentAccessController = (ConcurrentAccessController) synCtx
                     .getProperty(SynapseConstants.SYNAPSE_CONCURRENT_ACCESS_CONTROLLER);
+            int available = concurrentAccessController.incrementAndGet();
+            int concurrentLimit = concurrentAccessController.getLimit();
             if (log.isDebugEnabled()) {
-                int available = concurrentAccessController.incrementAndGet();
-                int concurrentLimit = concurrentAccessController.getLimit();
                 log.debug("Concurrency Throttle : Connection returned" + " :: " + +available + " of available of"
```
Comment (harsha89) [inline modules/core/src/main/java/org/apache/synapse/util/ConcurrencyThrottlingUtils.java]: There are two + signs and space is missing at the end of "of available of". Please correct them as well

Comment (GDLMadushanka) [reply modules/core/src/main/java/org/apache/synapse/util/ConcurrencyThrottlingUtils.java]: +1 Done

Comment (harsha89) [reply modules/core/src/main/java/org/apache/synapse/util/ConcurrencyThrottlingUtils.java]: @GDLMadushanka you might have missed adding " of available of" space and make this " of available of " ?

**Pull request merged**